tests/e2e/e2e_setup_test.go

```diff
@@ -171,6 +179,17 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		}
 	}
 
+	if str := os.Getenv(forkHeightEnv); len(str) > 0 {
+		forkHeight64, err = strconv.ParseInt(str, 0, 64)
+		s.forkHeight = int(forkHeight64)
+		s.Require().NoError(err)
+		s.isFork = true
+
+		if s.forkHeight != 0 {
+			s.T().Log(fmt.Sprintf("%s was set to height %v", forkHeightEnv, s.forkHeight))
+		}
```

Comment:
I think we should always print the height since someone might provide 0 as the value of the `forkHeightEnv`
```suggestion
		s.T().Log(fmt.Sprintf("fork upgrade is enabled, %s was set to height %v", forkHeightEnv, s.forkHeight))
```